contracts/mainnet/connectors/instaLite/main.sol

```diff
@@ -0,0 +1,115 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+/**
+ * @title InstaLite Connector
+ * @dev Supply and Withdraw
+
+ */
+import { TokenInterface } from "../../common/interfaces.sol";
+import { Basic } from "../../common/basic.sol";
+import { Events } from "./events.sol";
+import { IInstaLite } from "./interface.sol";
+
+abstract contract InstaLiteConnector is Events, Basic {
+	/**
+	 * @dev Supply
+	 * @notice Supply eth/weth/stEth tokens into instalite.
+	 * @param vaultAddress Address of instaLite Contract.
+	 * @param token The address of token to be supplied.
+	 * @param amt The amount of token to be supplied.
+	 * @param to The address of the account on behalf of you want to supplied.
+	 * @param getId ID to retrieve amt.
+	 * @param setIds ID stores the amount of token deposited.
+	 */
+	function supply(
+		address vaultAddress,
+		address token,
+		uint256 amt,
+		address to,
```

Comment:
@bhavik-m @Vaibhav20201 To will be DSA address only. I.e `address(this)`

Reply:
I have updated the codebase